open-vm-tools/lib/include/escBitvector.h

```diff
@@ -139,7 +139,11 @@ static INLINE int EscBitVector_Test(EscBitVector const *bv, int n)
    ASSERT(n>=0 && n<ESC_BITVECTOR_SIZE);
 #ifdef __GNUC__
    {
+#ifdef VM_ARM_64
```

Comment:
given the x86 assembly code below, I think  we'd better gate that part under VM_X86_ANY and let the arm code fall back to use the C part.

Reply:
I've done some more digging into this section of code. Only one file makes reference to this header, and the code that actually used this... was removed in 2008.

This appears to be a dead file not even doing anything other than throwing compiler errors at me, that I put that hackish #ifdef into to avoid. I think it would be best to just remove it at this point.

https://github.com/vmware/open-vm-tools/commit/ef4a8b2d05b1c15bbe6337f3c36b1ff45402b624#diff-5066e7d95c24c4e592c9eb6e82d61655014eee22eed34c379fa4c1957cbbf387